aws/rust-runtime/aws-endpoint/src/partition/mod.rs

```diff
@@ -51,7 +51,7 @@ impl ResolveAwsEndpoint for PartitionResolver {
 
 #[derive(Debug)]
 pub struct Partition {
-    id: &'static str,
+    _id: &'static str,
```

Comment:
Clippy was complaining about this locally and the field seemed unused so I added the underscore